### ISSUE TITLE
[BUG] Messages sent from the Signal desktop client aren't displayed in Axolotl

### ISSUE THREAD

**Issue by Flaburgan** [nanu-c/axolotl#110]
### Description

Messages sent from Signal desktop client aren't displayed in Axolotl, so the conversation is hard to follow.

### Steps to Reproduce

1. Send a message to a contact from Signal Desktop client
2. Open Axolotl
3. Open the conversation with the contact

**Expected behavior:** [What you expect to happen]
The message sent from the Signal desktop client should be displayed in Axolotl conversation

**Actual behavior:** [What actually happens]
The message isn't displayed

#### Versions
0.7.7.2

#### Device
Fairphone 2

I should add that I use the desktop client on several computers and messages sent with one desktop client are displayed on other desktop clients so this is a problem on Axolotl side, not Signal.

**Comment by Flaburgan** [comment]
Sorry, I inverted the direction here. Messages sent from Axolotl are displayed in the desktop Signal client. It's messages sent from the desktop client that aren't pulled by Axolotl. I'm going to edit the issue.

**Comment by selfbc** [comment]
If this helps, it's from this commit: https://github.com/signalapp/Signal-Desktop/commit/7ca0dfdfbe91f8ed47e3feb7c5633375a5791602#diff-0b3c72f8fa053c9f175d037847bb9b32R124

I can give more information as needed.

**Comment by Flaburgan** [comment]
It looks like it only affects the current thread. So the new steps to reproduce would be:
- Open a conversation with a contact on Axolotl
- Switch off internet connection on your phone
- Send a message to your contact from Signal desktop
- Your contact answers you, his / her message is displayed on Signal desktop
- Switch on internet connection on your phone

Actual result:  
The message sent by your contact is received on Axolotl, but not the one you sent from the signal desktop client

Expected result:
Every messages are received by Axolotl.
